docs/sphinx/examples/python/tutorials/vqe.ipynb

```diff
@@ -40,30 +40,28 @@
    ],
    "source": [
     "import cudaq\n",
+    "import cudaq.lib\n",
     "\n",
     "geometry = [('H', (0., 0., 0.)), ('H', (0., 0., .7474))]\n",
-    "\n",
     "molecule, data = cudaq.chemistry.create_molecular_hamiltonian(\n",
     "    geometry, 'sto-3g', 1, 0)\n",
     "\n",
     "electron_count = data.n_electrons\n",
     "qubit_count = 2 * data.n_orbitals\n",
     "\n",
-    "kernel, angles = cudaq.make_kernel(list)\n",
-    "qubits = kernel.qalloc(qubit_count)\n",
-    "\n",
-    "# Prepare the Hartree Fock State.\n",
-    "kernel.x(qubits[0])\n",
-    "kernel.x(qubits[1])\n",
+    "@cudaq.kernel(jit=True)\n",
+    "def kernel(qubit_count: int, angles:list[float]):\n",
+    "    qubits = cudaq.qvector(qubit_count)\n",
     "\n",
-    "# Adds parameterized gates based on the UCCSD ansatz.\n",
-    "cudaq.kernels.uccsd(kernel, qubits, angles, electron_count, qubit_count)\n",
+    "    # Prepare the Hartree Fock State.\n",
+    "    x(qubits[0])\n",
+    "    x(qubits[1])\n",
     "\n",
-    "parameter_count = cudaq.kernels.uccsd_num_parameters(electron_count,\n",
-    "                                                     qubit_count)\n",
+    "    # Adds parameterized gates based on the UCCSD ansatz.\n",
+    "    chemistry.uccsd(qubits, angles, electron_count, qubit_count)\n",
```

Comment:
The compiler is unhappy if this is cudaq.lib.chemistry.uccsd instead of chemistry.uccsd.